jupyterhub/README.md

```diff
@@ -94,6 +94,10 @@ Contains build chain manifest for CUDA 11.0.3 enabled ubi 8 based images with py
 
 *NOTE:* Builds in this overlay require 4-6 GB of memory
 
+#### [odh-ray-cluster](notebook-images/overlays/odh-ray-cluster/)
+
+Contains deployment manifests for Ray setup with ODH jupyterhub.
```

Comment:
I think with our move towrads kfnbc as the default notebook provider we should probably gear Ray towards nbc as well. @crobby  / @LaVLaS  wdyt?

Reply:
@anishasthana @LaVLaS 

Currently we are using a Single User Profile as part of our Ray /Jupyterhub deployment that spawns a Ray cluster for the user when they start the "ray notebook image" from the spawner page.  Is it possible to do something similar with KFNBC? 

Is there an analogues spawner page UI with KFNBC right now? or is it still done by `oc apply`?